src/components/NewActivityList.js

```diff
@@ -76,6 +140,7 @@ class NewActivityList extends Component {
       money,
       how_long,
       userUid,
+      timeWorkedBefore,
```

Comment:
let's refactor these 2 requests to firebase
there are 2 ways
1) you want to send 2nd request after first is saved - use .then() then
2) you want to save at the same time, then we can do it in 1 request, 
https://firebase.google.com/docs/database/web/read-and-write#updating_or_deleting_data